server/repository/assessmentCycle/descriptions/update.ts

```diff
@@ -0,0 +1,34 @@
+import { CountryIso } from '@core/country'
+import { Objects } from '@core/utils'
+
+import { Assessment, Cycle } from '@meta/assessment'
+import { CommentableDescriptionName } from '@meta/assessment/commentableDescription'
+
+import { BaseProtocol, DB, Schemas } from '@server/db'
+
+export const update = async (
+  props: {
+    countryIso: CountryIso
+    assessment: Assessment
+    cycle: Cycle
+    content: string
+    sectionName: string
+    name: CommentableDescriptionName
+  },
+  client: BaseProtocol = DB
+): Promise<string> => {
+  const { countryIso, assessment, cycle, sectionName, name, content } = props
+  const schemaCycle = Schemas.getNameCycle(assessment, cycle)
+
+  // insert new | on conflict update
+  const query = `
```

Comment:
Do we need an insert if it doesn't exist? You can handle it in one query with 'on conflict' and we can call the method upsert ?

Reply:
Doesn't this do this?